model/Storage.inc.php

```diff
@@ -255,54 +255,25 @@ public static function getUploadInfo($key) {
 		}
 		return $info;
 	}
-
-	public static function getUploadQueueItemsAndInfo($hash) {
-		$sql = "SELECT * FROM storageUploadQueue s1 
-					JOIN (
-							SELECT uploadKey, MAX(time) AS time
-							FROM storageUploadQueue
-							GROUP BY userID, itemLibraryID, itemKey
-						 ) AS s2
-				    ON s1.uploadKey = s2.uploadKey
-					WHERE hash=?";
-
-		$uploads = Zotero_DB::query($sql, $hash);
-
-		$results = [];
-		foreach ($uploads as $upload) {
-			echo $upload['itemLibraryID']. ' '.$upload['itemKey'];
-
+	
+	public static function getRecentUploads($hash) {
+		$sql = "SELECT * FROM storageUploadQueue s1 JOIN ( 
+					SELECT uploadKey, MAX(time) AS time
+					FROM storageUploadQueue
+					GROUP BY userID, libraryID, itemKey) AS s2
```

Comment:
You have `key` in the schema file, not `itemKey`. I'm fine with just `key`, since we have other unprefixed item properties there.